boxr.gemspec

```diff
@@ -28,6 +28,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "awesome_print", "~> 1.8"
   spec.add_development_dependency "lru_redux", "~> 0.8"
   spec.add_development_dependency "parallel", "~> 1.0"
+  spec.add_development_dependency "rubyzip", "~> 1.0"
```

Comment:
Why rubyzip 1.x instead of the latest [2.3.2](https://github.com/rubyzip/rubyzip/releases/tag/v2.3.2) release?

https://github.com/rubyzip/rubyzip/releases

Reply:
No reason, my mistake. Fixed it.